examples/todoapp/src/generatedNoCheck/FoundryClient.ts

```diff
@@ -0,0 +1,13 @@
+import type { Auth, FoundryClientOptions } from '@osdk/legacy-client';
+import { BaseFoundryClient } from '@osdk/legacy-client';
+import { Ontology } from './Ontology';
+
+export class FoundryClient<TAuth extends Auth = Auth> extends BaseFoundryClient<typeof Ontology, TAuth> {
+  constructor(options: FoundryClientOptions<TAuth>) {
+    super(options, Ontology);
+  }
+
+  get ontology(): Ontology {
+    return super.ontology;
+  }
```

Comment:
Do we expect users to add things to this class? Otherwise why not just rename `BaseFoundryClient` and use that directly?

Reply:
Generated. We should fix the generator code if we want that.